.github/workflows/build.yml

```
@@ -28,62 +28,21 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  # Fetch image names built from [Bazzite repo](https://github.com/ublue-os/bazzite)
-  # and filter out certain ones.
-  generate_image_matrix:
-    permissions:
-      packages: read
-    runs-on: ubuntu-24.04
-    outputs:
-      BASE_IMAGES: ${{ steps.fetch_names.outputs.BASE_IMAGES }}
-    steps:
-      - id: fetch_names
-        shell: bash
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          org: ${{ env.SOURCE_ORG }}
-          repo: ${{ env.SOURCE_REPO }}
-        run: |
-          mapfile -t images < <(
-            gh api \
-              -H "Accept: application/vnd.github+json" \
-              -H "X-GitHub-Api-Version: 2022-11-28" \
-              --paginate \
-              "/orgs/${org}/packages?package_type=container" | \
-            jq -r \
-              ".[] | select(.repository.full_name == \"$org/$repo\") | .name" \
-          )
-
-          # Here we filter out images
-          _re='ally|surface|budgie|asus|cosmic'
-          images_allowed=()
-          for img in "${images[@]}"; do
-            [[ $img =~ $_re ]] && continue
-            images_allowed+=($img)
-          done && unset -v images _re img
-          images_allowed=( $(echo "${images_allowed[@]}" | sort -u) ) # Ensure we have no repetition
-          
-          # Create the json output
-          images_json="[ "
-          for img in "${images_allowed[@]}"; do
-            images_json+="\"$img\", "
-          done && unset -v img images_allowed
-          images_json="${images_json%%,@( )} ]"
-          
-          echo "BASE_IMAGES=$images_json" | tee --append $GITHUB_OUTPUT
 
   build_push:
-    needs: [generate_image_matrix]
     name: Build and push image
     runs-on: ubuntu-24.04
 
     strategy:
       matrix:
```

```suggestion
    strategy:
      fail-fast: false
      matrix:
```
Apparently we need fail-fast to be false, seen this on most of our repos